client/ayon_core/hosts/maya/api/lib.py

```diff
@@ -2640,13 +2640,14 @@ def reset_scene_resolution():
     Returns:
         None
     """
-
-    folder_attributes = get_current_project_folder()["attrib"]
+    project_name = get_current_project_name()
+    project_entity = ayon_api.get_project(project_name)
+    project_attribs = project_entity["attrib"]
 
     # Set resolution
-    width = folder_attributes.get("resolutionWidth", 1920)
-    height = folder_attributes.get("resolutionHeight", 1080)
-    pixelAspect = folder_attributes.get("pixelAspect", 1)
+    width = project_attribs.get("resolutionWidth", 1920)
+    height = project_attribs.get("resolutionHeight", 1080)
+    pixelAspect = project_attribs.get("pixelAspect", 1)
```

Comment:
Pretty sure this change is wrong. It shouldn't reset to the project, but to the task attributes.

Reply:
do you mean the pseudo code below?
```
task_entity = ayon_api.get_task_by_name(
            project_name, folder_entity["id"], task_name
        )
task_attributes = task_entity["attrib"]
width = task_attributes .get("resolutionWidth", 1920)
height = task_attributes .get("resolutionHeight", 1080)
pixelAspect = task_attributes .get("pixelAspect", 1)
```